src/ILLink.RoslynAnalyzer/DiagnosticCategory.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace ILLink.RoslynAnalyzer
+{
+	internal static class DiagnosticCategory
+	{
+		public const string Publish = nameof (Publish);
```

Comment:
Why is this "publish" - a very general term - and we use it for single-file
And then we have "trimming" - which is specific just for trimming.

Shouldn't it be "single-file" and "trimming" instead?

Reply:
True. I've changed the category to SingleFile.